linter/whitelists/de.txt

```diff
@@ -311,6 +311,7 @@ production
 Produktkatalogen
 Programmierkenntnisse
 Projects
+Protokoll
```

Comment:
You have added `Protokoll`, however you do not use that word. Also that word probably does not need to be on the German whitelist? ;)